src/transformers/configuration_utils.py

```diff
@@ -480,11 +480,19 @@ def from_pretrained(cls, pretrained_model_name_or_path: Union[str, os.PathLike],
                 'http://hostname': 'foo.bar:4012'}.` The proxies are used on each request.
             use_auth_token (`str` or *bool*, *optional*):
                 The token to use as HTTP bearer authorization for remote files. If `True`, will use the token generated
```

Comment:
```suggestion
                The token to use as HTTP bearer authorization for remote files. If `True`, or not specified, will use the token generated
```

I would find it clearer to just update the wording here, but it's a nitpick